client-templates/nexus/.env.sample

```diff
@@ -18,12 +18,18 @@ ACCEPT=accept.json
 # The path for the broker's internal healthcheck URL. Must start with a '/'.
 BROKER_HEALTHCHECK_PATH=/healthcheck
 
-# Provide RES_BODY_URL_SUB with the URL of the Nexus without credentials and http protocol
+# Provide RES_BODY_URL_SUB with the URL of the Nexus without credentials
 # This URL substitution is required for NPM integration
-# RES_BODY_URL_SUB=http://<your.nexus.hostname>/repository
+# RES_BODY_URL_SUB=https://<your.nexus.hostname>/repository
 
 # Nexus validation url, checked by broker client systemcheck endpoint
 # readonly users must have 'nx-metrics-all' privilege enabled to access
 # https://support.sonatype.com/hc/en-us/articles/226254487-System-Status-and-Metrics-REST-API
+
+# NOTE! If NOT using username and password in BASE_NEXUS_URL, because your Nexus server is open on a private network, then use '/service/rest/v1/status' for BROKER_CLIENT_VALIDATION_URL.
+# '/service/rest/v1/status/check' requires an authenticated connection while '/service/rest/v1/status' does not.
+# Need to make sure the accept.json also accounts for this endpoint
```

Comment:
Im going to leave a note / example in the README update PR for GA

Reply:
or shall we leave both `/service/rest/v1/status` and `/service/rest/v1/status/check` in the `accept.json`?

Reply:
@gitphill ^

Reply:
holding off updating https://github.com/snyk/broker/pull/417 until we come to a conclusion here

Reply:
is `/service/rest/v1/status/*` ok?

Reply:
ok, lets list both